src/main/java/wooteco/subway/config/WebMvcConfig.java

```diff
@@ -4,34 +4,25 @@
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import wooteco.subway.web.member.LoginMemberMethodArgumentResolver;
-import wooteco.subway.web.member.interceptor.BasicAuthInterceptor;
 import wooteco.subway.web.member.interceptor.BearerAuthInterceptor;
-import wooteco.subway.web.member.interceptor.SessionInterceptor;
 
 import java.util.List;
 
 @Configuration
 public class WebMvcConfig implements WebMvcConfigurer {
-    private final BasicAuthInterceptor basicAuthInterceptor;
-    private final SessionInterceptor sessionInterceptor;
     private final BearerAuthInterceptor bearerAuthInterceptor;
     private final LoginMemberMethodArgumentResolver loginMemberArgumentResolver;
 
-    public WebMvcConfig(BasicAuthInterceptor basicAuthInterceptor,
-                        SessionInterceptor sessionInterceptor,
-                        BearerAuthInterceptor bearerAuthInterceptor,
-                        LoginMemberMethodArgumentResolver loginMemberArgumentResolver) {
-        this.basicAuthInterceptor = basicAuthInterceptor;
-        this.sessionInterceptor = sessionInterceptor;
+    public WebMvcConfig(BearerAuthInterceptor bearerAuthInterceptor, LoginMemberMethodArgumentResolver loginMemberArgumentResolver) {
         this.bearerAuthInterceptor = bearerAuthInterceptor;
         this.loginMemberArgumentResolver = loginMemberArgumentResolver;
     }
 
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
-        registry.addInterceptor(basicAuthInterceptor).addPathPatterns("/me/basic");
-        registry.addInterceptor(sessionInterceptor).addPathPatterns("/me/session");
-        registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/me/bearer");
+         registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/members");
+         registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/members/*");
+         registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/members/*/*");
```

Comment:
```suggestion
         registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/members/**");
```

![image](https://user-images.githubusercontent.com/40811999/83027424-13104680-a06b-11ea-8417-afc260345e2c.png)

패턴을 확인해주세요~